src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -77,6 +77,23 @@ private static string GetBitcoinCorePath(string version)
             throw new FileNotFoundException($"Could not load the file {path}.");
         }
 
+        private static string GetStratisXPath(string version)
+        {
+            string path;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                path = $"../../../../External libs/StratisX/{version}/Windows/stratisd.exe";
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
That's definitely gonna fail for Linux and Mac so in the meantime, can you surround the inside of your test with ` if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
            {`